packages/conform-react/context.tsx

```diff
@@ -83,7 +71,7 @@ type SubfieldMetadata<
 	Schema,
 	FormSchema extends Record<string, any>,
 	FormError,
-> = [Schema] extends [Primitive]
+> = [Schema] extends [Primitive | Date | File]
```

Comment:
@edmundhung this seems to be the only place Primitive is being used, whats the intent for overloading the term "Primitive" here with `Date` and `File`?

maybe another term like `FormValue` or something along those lines might be better here